packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -35,7 +35,10 @@ interface GridCellProps {
    * Please use onCellAction at the collection level instead.
    * @deprecated
    **/
-  onAction?: () => void
+  onAction?: () => void,
+  // TODO: perhaps make this a hidden prop
+  /** Whether the cell children should never recieve focus. Makes the cell recieve focus instead if the cell child were to receive focus. */
+  preventChildrenFocus?: boolean
```

Comment:
If we don't allow focusable children at all, then do we even need to add this prop? The Card component itself could take care of verifying this and emitting an error/warning. The checkbox should get `tabIndex="-1"` so it isn't tabbable also.

Reply:
So the reason I had to do this is because the focusable tree walker below doesn't have the `tabbable: true` option and thus the checkbox is regarded as the next focusable child when pressing the right arrow. I can't add the `tabbable: true` option because it would then break navigation for ActionGroups inside grid rows since those have a roving tab index and thus the other buttons in the group would be completely skipped when pressing the right arrow

Reply:
Hmm so, I think it _should_ be `tabbable: true`. Ideally the action group would handle its own keyboard navigation, rather than the grid handling it. But not sure how it would delegate back to the grid when reaching the end rather than wrapping back to the start of the action group... 🤔

Reply:
Right, this feels like the issue "Edit Mode" was trying to fix since we would essentially be disabling grid keyboard navigation and letting the child components handle keyboard nav